manifests/certonly.pp

```diff
@@ -45,6 +45,7 @@
   Letsencrypt::Plugin                       $plugin               = 'standalone',
   Array[Stdlib::Unixpath]                   $webroot_paths        = [],
   String[1]                                 $letsencrypt_command  = $letsencrypt::command,
+  Integer[2048]                             $key_size             = $letsencrypt::key_size,
```

Comment:
should we enforce 2048, 4096 and 8192 as only allowed values?

Reply:
I considered that, but 3072 is also used by some. Not sure how much we should limit it here

Reply:
ah true, let's stick with 2048 as a lower limit.